doc/changes/latest.inc

```diff
@@ -86,6 +86,8 @@ Bugs
 
 - Fix channel grouping error when using "butterfly mode" with :meth:`mne.io.Raw.plot` (:gh:`10087` by `Daniel McCloy`_)
 
+- Fix inconsistent behavior of ``mne.preprocessing.annotate_*`` functions: All return :class:`mne.Annotations` objects with the ``orig_time`` attribute set to ``None``(:gh:`10067` by `Stefan Appelhoff`_)
```

Comment:
I screwed this up during rebasing: this is an old version (AND with wrong syntax) ... needs to be fixed (will do tomorrow or so)